applications/serial_lte_modem/src/http_c/slm_at_httpc.c

```diff
@@ -67,6 +67,7 @@ extern struct at_param_list at_param_list;
 extern uint8_t data_buf[SLM_MAX_MESSAGE_SIZE];
 
 static struct k_thread httpc_thread;
+static bool httpc_in_thread;
```

Comment:
I think you can use k_thread_join() to decide above httpc_thread is running or not, instead of adding a new global static.

Reply:
k_thread_join can't be used to check if thread is created or not.

Reply:
Try to poll the `k_tid_t` returned from `k_thread_create`.
If 0, thread is not running or already exited
If -EAGAIN, thread is running